noodles-bed/src/de.rs

```diff
@@ -0,0 +1,939 @@
+use std::fmt;
+use std::io::BufRead;
+use std::ops::{AddAssign, MulAssign, Neg};
+
+use noodles_core::Position;
+
+use serde::Deserialize;
+use serde::de::{
+    self, DeserializeSeed, EnumAccess, IntoDeserializer, MapAccess, SeqAccess,
+    VariantAccess, Visitor,
+};
+
+use crate::{Record, reader::Reader, error::{Error, Result}};
+
+impl<'de> Deserialize<'de> for Record<3> {
```

Comment:
This is open for discussion, but I feel like `'a` is the conventional lifetime variable to use, and you only need unique names like this when multiple lifetimes are present.

Reply:
Just nitpick but doesn't seem to be the case on SerDe, its documentation states `'de` for deserializer-related traits, therefore I'd follow its conventions... `'a` is for more generic (canonical/conventional) as seen elsewhere.

Reply:
Indeed, that's fair.